### PR TITLE
precompile gas calculation

### PR DESCRIPTION
**Pull request by ermalkaleci**

### CODE REVIEW

**Review by xlc — APPROVED** [review]

File: runtime/common/src/precompile/mock.rs [hunk 1]
```diff
@@ -572,13 +574,72 @@ frame_support::construct_runtime!(
...
+				balance: 1000 * crate::dollar(ACA),
+				storage: BTreeMap::new(),
+				code: vec![],
+				enable_contract_development: true,
+			};
+			accounts.insert(dev_acc, account);
+		}
+
+		accounts
+	}
```
Comment (zjb0807) [inline runtime/common/src/precompile/mock.rs:641]: why not use `use acala_service::chain_spec::mandala::evm_genesis;` here?

Comment (ermalkaleci) [reply runtime/common/src/precompile/mock.rs:641]: @zjb0807 dependency loop

Comment (ermalkaleci) [reply runtime/common/src/precompile/mock.rs:641]: > Maybe can move `evm_genesis` into `acala_primitives`

agree

Comment (ermalkaleci) [reply runtime/common/src/precompile/mock.rs:641]: runtime-common actually

**Review by zjb0807 — APPROVED** [review]

**Pull request merged**